src/Symfony/Contracts/Cache/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "symfony/cache-contracts",
+    "type": "library",
+    "description": "Generic abstractions related to caching",
+    "keywords": ["abstractions", "contracts", "decoupling", "interfaces", "interoperability", "standards"],
```

Comment:
It would be redundant with words in the name of the package.

Reply:
packagist allows filtering search result by keywords, so that would still be useful IMO.

Reply:
> packagist allows filtering search result by keywords, so that would still be useful IMO.

wow, I just played with that, we now need to revisit all our composer.json files!